### PR TITLE
Update new course page fields

### PR DESCRIPTION
**Pull request by bderusha**

### CODE REVIEW

File: course_discovery/apps/publisher/forms.py [hunk 1]
```diff
@@ -66,13 +66,37 @@ class CustomCourseForm(CourseForm):
...
     )
 
+    program_type = forms.ChoiceField(
+        label=_('Choose a program'),
+        choices=(('micromasters', _('MicroMasters')), ('xseries', _('XSeries'))),
+        required=False
+    )
+
+    primary_subject = forms.ModelChoiceField(
+        queryset=Subject.objects.all(),
```
Comment (awais786) [inline course_discovery/apps/publisher/forms.py]: This query can be called once and store in a var and use at all three places.

File: course_discovery/templates/publisher/add_course_form.html [hunk 2]
```diff
@@ -213,6 +240,7 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
                     <div class="col col-6">
                         <label class="field-label ">{{ course_form.short_description.label_tag }}</label>
                         {{ course_form.short_description }}
+                        <p>140 character limit, including spaces.</p>
```
Comment (awais786) [inline course_discovery/templates/publisher/add_course_form.html]: 140 limit need to be enforce at form level.

File: course_discovery/templates/publisher/add_course_form.html [hunk 3]
```diff
@@ -234,6 +262,7 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
                     <div class="col col-6">
                         <label class="field-label ">{{ course_form.full_description.label_tag }}</label>
                         {{ course_form.full_description }}
+                        <p>2500 character limit, including spaces.</p>
```
Comment (awais786) [inline course_discovery/templates/publisher/add_course_form.html]: 2500 limit need to be enforce also at form level.

File: course_discovery/templates/publisher/add_course_form.html [hunk 4]
```diff
@@ -342,6 +357,20 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
                     <div class="col col-6">
                         <label class="field-label ">{{ course_form.prerequisites.label_tag }}</label>
                         {{ course_form.prerequisites }}
+                        <p>200 character limit, including spaces.</p>
```
Comment (awais786) [inline course_discovery/templates/publisher/add_course_form.html]: 200 limit need to be enforce also at form level.

**Review by awais786 — CHANGES_REQUESTED** [review]

File: course_discovery/templates/publisher/add_course_form.html (hunk 2 above)
Comment (bderusha) [reply course_discovery/templates/publisher/add_course_form.html]: It is.

File: course_discovery/templates/publisher/add_course_form.html (hunk 3 above)
Comment (bderusha) [reply course_discovery/templates/publisher/add_course_form.html]: it is. :)

File: course_discovery/templates/publisher/add_course_form.html (hunk 4 above)
Comment (bderusha) [reply course_discovery/templates/publisher/add_course_form.html]: it is! :D

**Comment by bderusha** [comment]
@awais786 @waheedahmed @tasawernawaz Please review

File: course_discovery/apps/publisher/forms.py [hunk 5]
```diff
@@ -77,13 +77,42 @@ class CustomCourseForm(CourseForm):
         label=_('Organization Course Admin'),
     )
 
+    # program_type = forms.MultipleChoiceField(
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/forms.py]: Remove this commented code if not needed.

File: course_discovery/templates/publisher/add_course_form.html [hunk 6]
```diff
@@ -169,11 +169,58 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
     <div class="card course-form">
         <div class="course-information">
             <fieldset class="form-group grid-container grid-manual">
+                <div class="field-title">{% trans "PROGRAM ASSOCIATION" %}</div>
+                <div class="row">
+                    <div class="col col-6 help-text">
+                        <ul>
+                            <li>{% trans 'Is this course a part of any programs?' %}</li>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/add_course_form.html]: Be consistent with single or double quotes.

File: course_discovery/templates/publisher/add_course_form.html [hunk 7]
```diff
@@ -213,6 +260,7 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
                     <div class="col col-6">
                         <label class="field-label ">{{ course_form.short_description.label_tag }}</label>
                         {{ course_form.short_description }}
+                        <p>140 character limit, including spaces.</p>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/add_course_form.html]: i18n support.

File: course_discovery/templates/publisher/add_course_form.html [hunk 8]
```diff
@@ -234,6 +282,7 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
                     <div class="col col-6">
                         <label class="field-label ">{{ course_form.full_description.label_tag }}</label>
                         {{ course_form.full_description }}
+                        <p>2500 character limit, including spaces.</p>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/add_course_form.html]: i18n support.

File: course_discovery/templates/publisher/add_course_form.html [hunk 9]
```diff
@@ -342,6 +377,20 @@ <h2 class="layout-title">{% trans "About page information" %}</h2>
                     <div class="col col-6">
                         <label class="field-label ">{{ course_form.prerequisites.label_tag }}</label>
                         {{ course_form.prerequisites }}
+                        <p>200 character limit, including spaces.</p>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/add_course_form.html]: i18n support.

**Review by waheedahmed — CHANGES_REQUESTED** [review]

**Comment by bderusha** [comment]
@awais786 @waheedahmed comments addressed

**Review by waheedahmed — APPROVED** [review]

**Comment by rlucioni** [comment]
Don't forget to delete merged branches!

**Pull request merged**